bin/mvn

```diff
@@ -22,6 +22,16 @@ if [ -z "$NONINTERACTIVE" ]; then
     INTERACTIVE_FLAGS="-it"
 fi
 
+if [[ -f ${GOOGLE_APPLICATION_CREDENTIALS} ]]; then
```

Comment:
My style preference is to always quote arguments that you don't need the shell to parse in any way, which is consistent with this file's current style.

```suggestion
if [[ -f "$GOOGLE_APPLICATION_CREDENTIALS" ]]; then
```